engine/src/main/resources/spring/aps/servicesConfig.xml

```diff
@@ -59,6 +59,8 @@
 
     <!-- === SERVICES === -->
 
+    <bean id="ComponentService" class="org.entando.entando.aps.system.services.component.ComponentService" />
```

Comment:
this is a new bean, time ago we discussed and decided to create a new bean with Java annotation if possible, isn't it possible?

Reply:
Yes, but all services are defined like this, and to make something homogeneous I would have had to change many parts; I think this type of refactoring should be done in special tickets